frontend/app_flowy/lib/workspace/application/settings/settings_location_cubit.dart

```diff
@@ -12,10 +12,22 @@ const String kSettingsLocationDefaultLocation =
 
 class SettingsLocation {
   SettingsLocation({
-    this.path,
-  });
+    String? path,
+  }) : _path = path;
 
-  String? path;
+  String? _path;
+
+  set path(String? path) {
+    _path = path;
+  }
+
+  String? get path {
+    if (Platform.isMacOS) {
+      // remove the prefix `/Volumes/Macintosh HD/Users/`
+      return _path?.replaceFirst('/Volumes/Macintosh HD/Users', '');
```

Comment:
Is `Macintosh HD` the same with different computer settings?

Reply:
Yes. If the user didn't change the name when reinstalling the system. It will be `Macintosh HD` always.

Reply:
So using regex: `volums/*/User.` would be better.

Reply:
Done, using `/Volumes/[^/]+` to match the prefix now.